platform/ui/src/views/StudyList/components/StudyListPagination.js

```diff
@@ -1,7 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
+import {
+  Typography,
+  ButtonGroup,
+  Button,
+  IconButton,
+} from '../../../components/';
 
-const StudyListPagination = () => {
-  return <div>{`<- PAGINATION CONTENT ->`}</div>;
+const StudyListPagination = props => {
+  const [currentPage, setCurrentPage] = useState(1);
```

Comment:
Following Danny's advices, shouldn't we be receiving the currentPage and setCurentPage from props, so we can basicaly display parent state of this component.